### PR TITLE
Publish to Maven Central from GitHub Actions

### PR DESCRIPTION
**Pull request by dmikurube**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -114,6 +114,7 @@ publishing {
                         // http://central.sonatype.org/pages/requirements.html#license-information
                         name = "The Apache License, Version 2.0"
                         url = "https://www.apache.org/licenses/LICENSE-2.0.txt"
+                        distribution = "repo"
```
Comment (hiroyuki-sato) [inline build.gradle:117]: Could you tell me what this line means? (For future development)

**Review by hiroyuki-sato — APPROVED** [review]
LGTM👍(I have a question)

File: build.gradle (hunk 1 above)
Comment (dmikurube) [reply build.gradle:117]: It is optional information in `pom.xml` to represent that the official location to distribute this library is Maven Central.

> distribution: This describes how the project may be legally distributed. The two stated methods are repo (they may be downloaded from a Maven repository) or manual (they must be manually installed).
>
> https://maven.apache.org/pom.html#licenses

**Comment by dmikurube** [comment]
Thanks!

File: build.gradle (hunk 1 above)
Comment (hiroyuki-sato) [reply build.gradle:117]: Thanks!

**Pull request merged**